tests/manual-test-cases/Group19-ROBO/19-3-ROBO-VM-Placement.md

```diff
@@ -0,0 +1,41 @@
+Test 19-3 - ROBO - VM Placement
+=======
+
+# Purpose:
+To verify that the VM placement feature specified works as expected in a vSphere ROBO Advanced environment without DRS.
+The current placement strategy is to avoid bad host selection, instead of selecting the "best" possible host.
+
+# References:
+1. [vSphere Remote Office and Branch Office](http://www.vmware.com/products/vsphere/remote-office-branch-office.html)
+2. [VM Placement without DRS](https://github.com/vmware/vic/issues/7282)
+
+# Environment:
+This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter creation. This test should be executed in the following topologies and should have vSAN enabled.
+* 1 vCenter host with 3 clusters, where 1 cluster has 1 ESXi host and the other 2 clusters have 3 ESXi hosts each
+* 2 vCenter hosts connected with ELM, where each vCenter host has a cluster/host/datacenter topology that emulates a customer environment (exact topology TBD)
+
+See https://confluence.eng.vmware.com/display/CNA/VIC+ROBO for more details.
+
+# Test Steps:
+1. Deploy a ROBO Advanced vCenter testbed for both environments above
+2. Install the VIC appliance on a particular (multi/single-host) cluster on vCenter
```

Comment:
The use of a single host cluster for this test would be:
a. to ensure we deal cleanly with that scenario
b. to check resource exhaustion behaviour in a simple setting.

We _must_ test in a multi-host cluster as that is what this placement logic is expressly for. These should be pulled out as two distinct variants of the test rather than incidentally noted in the current manner.

It's okay to leave comments in for the multi-host test saying we're not sure of exact behaviour at this time as it will be contingent on the algorithm design for the placement, but we should note the we are able to reach whatever cluster utilization level we would expect from the cVM size, cluster capacity and placement logic.